### ISSUE TITLE
[BUG] Running test suits displays incorrect failure

### ISSUE THREAD

**Issue by AlexanderEggers** [mobile-dev-inc/maestro#615]
When running multiple tests instead of a single flow, the CLI always returns with failures but the tests are actually passing and running fine when watching the simulator. I followed the documentation mentioned here: https://maestro.mobile.dev/cli/test-suites-and-reports 

![Screenshot 2023-01-15 at 6 22 23 pm](https://user-images.githubusercontent.com/7427545/212528273-e20aabb1-0a76-4af1-81bc-bacd5b8f47fb.png)

When running that specific flow alone, there are no errors and the test runs fine.

**Comment by Leland-Takamine** [comment]
Hey @AlexanderEggers this is likely due to having your subflows at the top-level directory. Maestro will attempt to run these as standalone flows. You can put your subflows in a subdirectory to prevent Maestro from running them as top-level flows.

**Comment by AlexanderEggers** [comment]
@Leland-Takamine That is what I did. All my subflows are in a separate folder.

**Comment by Leland-Takamine** [comment]
@AlexanderEggers Would you be able to share a screenshot of your directory structure along with the flow contents. We'll try to repro on our end

**Comment by AlexanderEggers** [comment]
@Leland-Takamine 

<img width="766" alt="Screenshot 2023-01-16 at 12 25 48 pm" src="https://user-images.githubusercontent.com/7427545/212579869-ea538d0d-413e-44f1-8674-0ae35e6a339b.png">

Flow content (01-login.yaml):

```yaml
appId: myappid.debug
---
- clearState
- clearKeychain
- launchApp
- runScript: scripts/getLoginFlowEnvVars.js
- runFlow: subflows/login.yaml
```

Flow content (login.yaml):

```yaml
appId: myappid.debug
---
- waitForAnimationToEnd:
    timeout: 5000
- tapOn: "Email"
- inputText: ${output.email}
- tapOn: "Password"
- inputText: ${output.password}
- tapOn: "Log In"
```

**Comment by dmitry-zaitsev** [comment]
This will be fixed in Maestro 1.19.1 (ETA: today)

**Comment by AlexanderEggers** [comment]
Awesome, thanks @dmitry-zaitsev 😃
